docker-compose.yaml

```
@@ -0,0 +1,41 @@
+      - DB_NAME=switchmap
+      - DB_USER=switchmap
+      - DB_PASSWORD=CHANGE_ME_NOW
+    depends_on:
+      mysql:
+        condition: service_healthy
+
+  poller:
+    build: .
+    command: ["bin/systemd/switchmap_poller", "--start"]
+    depends_on:
+       - server
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 7

(indentation)